pep-0682.rst

```diff
@@ -134,7 +134,10 @@ where ``z`` is allowed for floating-point presentation types (``f``, ``g``,
 etc.,  as defined by the format specification documentation).  Support for
 ``z`` is provided by the ``.__format__()`` method of each numeric type,
 allowing the specifier to be used in f-strings, built-in ``format()``, and
-``str.format()``.  The %-formatting style will not support the new option.
+``str.format()``.
+
+When ``z`` is present, negative zero (whether the original value or the
+result of rounding) will be normalized to positive zero.
```

Comment:
Additionally, I removed mention of %-formatting here-- it seems enough to mention it below in the notes.